docs/recipes/plugin/lighthouse-plugin-preload-as/preload-as-audit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../../lighthouse-core/index.js').Audit;
```

Comment:
a plugin would usually be written outside of our repo.. so wanna put the typical require path here?

Reply:
> a plugin would usually be written outside of our repo.. so wanna put the typical require path here?

yeah, I'll switch it. We do want it runnable (I was thinking about adding a `recipes` smokehouse target), but we can worry about that later